### PR TITLE
Improve mapping of metric names to InfluxDB field names

### PR DESCRIPTION
**Pull request by afausti**

### CODE REVIEW

File: examples/influxdb_data_model.ipynb [hunk 1]
```diff
@@ -36,7 +36,7 @@
     "\n",
     "* lsst.verify package -> InfluxDB measurement\n",
     "* lsst.verify metadata -> InfluxDB tag\n",
-    "* lsst.verify metric value - > InfluxDB field\n",
+    "* lsst.verify metric -> InfluxDB field\n",
```
Comment (kfindeisen) [inline examples/influxdb_data_model.ipynb]: Should this line say "InfluxDB field key" (if I understand the [InfluxDB terminology](https://docs.influxdata.com/influxdb/v1.7/concepts/key_concepts/#discussion) correctly), and should there be another line saying `lsst.verify metric value -> InfluxDB field value`?

Comment (kfindeisen) [reply examples/influxdb_data_model.ipynb]: Also, please tell me this notebook isn't the primary documentation for the translations...

File: examples/influxdb_data_model.ipynb [hunk 2]
```diff
@@ -48,8 +48,9 @@
     "`lsst.verify` adds metadata to the verification jobs uploaded to SQuaSH. The mapping of these metadata to either InfluxDB tags or fields is defined here:\n",
     "\n",
     "1. By default, if the SQuaSH metadata key is not found in the mapping, the original key will be preserved and it will be written to InfluxDB as a tag.\n",
-    "2. You can use the mapping to rename the metadata key when appropriate. \n",
-    "3. Finally, if `schema` is set to `None`, then the corresponding metadata won't be written to InfluxDB."
+    "2. You can use the mapping to rename a metadata key when appropriate. \n",
+    "3. `schema` defines if the metadata is mapped to an InfluxDB tag or field. If `schema` is set to `None`, then the metadata is dropped and won't be written to InfluxDB.\n",
+    "4. Finally, it is also possible to define a transformation on the streaming data."
```
Comment (kfindeisen) [inline examples/influxdb_data_model.ipynb:62]: I think point 1 would be easier to understand after point 3. Maybe 3, 2, 1?

File: examples/influxdb_data_model.ipynb [hunk 3]
```diff
@@ -136,8 +137,8 @@
...
     "The rationale for this mapping is the following:\n",
-    "1. `ci_id`, `run_id` and `squash_id` are sequential ids. `ci_url` and `run_id_url` are different for each run. The reason for adding them as InfluxDB fields is to [reduce InfluxDB series cardinality](https://docs.influxdata.com/influxdb/v1.7/concepts/schema_and_data_layout/#discouraged-schema-design) (DM-18342) and we also want to display these information in the dashboards.\n",
-    "2. We define a common datamodel mapping variables from different execution environments example `ci_id` and `run_id` to `run_id`, `ci_url` and `run_id_url` to `run_url`, `ci_dataset` and `dataset` to `dataset`. The corresponding values can be filtered by the `env_name` tag. \n",
+    "1. The reason for mapping `ci_id`, `run_id`, `squash_id`, `ci_url` and `run_id_url` to InfluxDB fields is to [reduce InfluxDB series cardinality](https://docs.influxdata.com/influxdb/v1.7/concepts/schema_and_data_layout/#discouraged-schema-design) `ci_id`, `run_id` and `squash_id` are sequential ids and `ci_url` and `run_id_url` are also different for each run (DM-18342). We also want to display these information in dashboards so they m.\n",
```
Comment (kfindeisen) [inline examples/influxdb_data_model.ipynb]: There's a missing period after "reduce InfluxDB series cardinality". The end of the line seems cut off.

File: examples/influxdb_data_model.ipynb [hunk 4]
```diff
@@ -457,11 +458,8 @@
     "               The request was processed successfully\n",
     "             400:\n",
     "               Malformed syntax or bad query\n",
-    "\n",
-    "        Note\n",
-    "        ----\n",
-    "        `lsst.verify` measurement and InfluxDB measurement mean different things. \n",
```
Comment (kfindeisen) [inline examples/influxdb_data_model.ipynb:463]: Why remove this? It's an important distinction.

File: examples/influxdb_data_model.ipynb [hunk 5]
```diff
@@ -513,16 +511,17 @@
     "    fields_by_measurement = {}\n",
     "    for verify_measurement in data['measurements']:\n",
     "        # DM-18360 - SQuaSH API /measurements should return the verification package \n",
-    "        influxdb_measurement = verify_measurement['metric'].split('.')[0]\n",
+    "        influxdb_measurement, field_key = verify_measurement['metric'].split('.')\n",
```
Comment (kfindeisen) [inline examples/influxdb_data_model.ipynb]: Do we need to worry about what happens if `verify_measurement['metric']` has more than one period?

File: examples/influxdb_data_model.ipynb [hunk 6]
```diff
@@ -557,15 +556,17 @@
     "\n",
     "for job_id in jobs['ids']:\n",
     "    \n",
-    "    data = requests.get(SQUASH_API_URL + \"/job/{}\".format(job_id)).json()\n",
+    "    if job_id > 5244:\n",
```
Comment (kfindeisen) [inline examples/influxdb_data_model.ipynb]: What's this magic number for?

**Review by kfindeisen — APPROVED** [review]
A few requests for clarification (and one potential bug) for the upload changes, but otherwise looks good. I'm afraid I don't understand the Travis/Docker changes at all, so I can't comment on those.

File: examples/influxdb_data_model.ipynb (hunk 1 above)
Comment (afausti) [reply examples/influxdb_data_model.ipynb]: No, this notebook is to exercise the mapping of `lsst.verify` jobs to InlfuxDB and it needs to be properly documented.

File: examples/influxdb_data_model.ipynb (hunk 4 above)
Comment (afausti) [reply examples/influxdb_data_model.ipynb:463]: I agree, it is an important distinction - it just feels like the note is not in the right place, will find a better place for it.

File: examples/influxdb_data_model.ipynb (hunk 5 above)
Comment (afausti) [reply examples/influxdb_data_model.ipynb]: Right, this needs to be improved. I think the right answer the SQuaSH API to return the package name (DM-18360) and remove it from the full qualified metric name.

File: examples/influxdb_data_model.ipynb (hunk 6 above)
Comment (afausti) [reply examples/influxdb_data_model.ipynb]: oops, should not have committed that line.

**Pull request merged**